src/presentation/controller/registration/Routing.ts

```diff
@@ -0,0 +1,53 @@
+import { PageRouting, PagesRouting } from "../PageRouting";
+import {
+  BASE_URL,
+  START_URL,
+  SUBMISSION_ID,
+  TRANSACTION_ID,
+} from "../../../config/constants";
+import RegistrationPageType from "./PageType";
+
+export const NAME_TEMPLATE = RegistrationPageType.name;
+export const NEXT_TEMPLATE = RegistrationPageType.next;
+
+export const NAME_URL = `${BASE_URL}/${NAME_TEMPLATE}`;
+
+export const NEXT_URL = `${BASE_URL}/transaction/${TRANSACTION_ID}/submission/${SUBMISSION_ID}/${NEXT_TEMPLATE}`;
+
+export const registrationRoutingStart = {
+  previousUrl: "/",
+  currentUrl: START_URL,
+  nextUrl: NAME_URL,
+  pageType: RegistrationPageType.start,
+};
+
+export const registrationRoutingName = {
+  previousUrl: START_URL,
+  currentUrl: NAME_URL,
+  nextUrl: NEXT_URL,
+  pageType: RegistrationPageType.name,
+};
+
+const registrationRoutingNext = {
+  previousUrl: NAME_URL,
+  currentUrl: NEXT_URL,
+  nextUrl: "/",
+  pageType: RegistrationPageType.next,
+};
+
+const list = [
+  registrationRoutingStart,
+  registrationRoutingName,
+  registrationRoutingNext,
+];
+
+export const registrationsRouting: PagesRouting = new Map<
```

Comment:
Would this work, if you populate the map on initialise:
`export const registrationsRouting: PagesRouting = new Map(
  list.map(routing => [routing.pageType, routing])
);`
then you wouldn't need the forEach

Reply:
Map is not an array but a key, value object
forEach adds the correct value with its corresponding key